src/Core/SettingsChangesHistory.cpp

```diff
@@ -98,6 +98,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"validate_enum_literals_in_operators", false, false, "A new setting"},
             {"allow_experimental_kusto_dialect", true, false, "A new setting"},
             {"allow_experimental_prql_dialect", true, false, "A new setting"},
+            {"h3togeo_lon_lat_result_order", false, false, "A new setting"},
```

Comment:
Let's change `false, false` to `true, false`. These bools are used by the compatibility mechanism in ClickHouse. The first value is the previous behavior, the second one is the new one.